install/kubernetes/cilium/values.yaml

```
@@ -435,9 +435,14 @@ enableK8sEventHandover: false
 # -- Enable CiliumEndpointSlice feature.
 enableCiliumEndpointSlice: false
 
-# -- Enable cilium ingress controller
-# This will automatically set enable-envoy-config as well.
-enableIngressController: false
+ingressController:
+  # -- Enable cilium ingress controller
+  # This will automatically set enable-envoy-config as well.
+  enabled: false
+
+  # -- EnforceIngressHttps enforces https for host having matching TLS host in Ingress.
+  # Incoming traffic to http listener will return 308 http error code with respective location in header.
+  enforceHttps: true
```

Nitpick: the actual option name differs

```suggestion
  # -- Enforce https for host having matching TLS host in Ingress.
  # Incoming traffic to http listener will return 308 http error code with respective location in header.
  enforceHttps: true
```

---

thanks, I wait for current runing CI jobs to complete and address the above.